### DISCUSSION TITLE
How do I download or generate G6 offline documents?

### DISCUSSION THREAD

**Discussion by addriumruss** [antvis/G6#6323] [Q&A]
### Describe the bug / 问题描述

请问怎么下载或生成G6离线文档

### G6 Version / G6 版本

🆕 5.x

### Operating System / 操作系统

Linux

### Browser / 浏览器

Chrome, Firefox

**Comment by Aarebecca** [comment]
1. git clone g6 项目
2. 使用 pnpm 安装依赖
3. 进入 `./packages/site` 目录
4. 执行 `pnpm build` 构建站点
5. 使用 `./dist` 目录下产物进行部署

**Comment by addriumruss** [comment]
您好 编译一堆错误，不知道是什么原因

  × Module not found: Can't resolve 'dumi/theme/slots/PreviewerActions'
   ╭─[..\..\node_modules\.pnpm\dumi@2.4.10_@babel+core@7.25.2_@swc+helpers@0.5.1_@types+node@20.16.5_@types+react@18.3.5_esl_rezk4qt3lbgj5ij43hz2lfycyy\node_modules\dumi\theme-default\builtins\Previewer\index.js:3:1]
 3 │ import { ReactComponent as IconError } from '@ant-design/icons-svg/inline-svg/filled/close-circle.svg';
 4 │ import classnames from 'classnames';
 5 │ import { useLiveDemo, useLocation } from 'dumi';
 6 │ import PreviewerActions from 'dumi/theme/slots/PreviewerActions';
   ·                              ───────────────────────────────────
 7 │ import React, { useRef } from 'react';
 8 │ import "./index.less";
 9 │ var Previewer = function Previewer(props) {
   ╰────


  × Module not found: Can't resolve 'dumi/theme/slots/CodeRunner'
    ╭─[..\..\node_modules\.pnpm\@antv+dumi-theme-antv@0.5.2_@algolia+client-search@4.24.0_@babel+core@7.25.2_@types+react@18._agqpr5sk7d6qtp7zumpftwkipe\node_modules\@antv\dumi-theme-antv\dist\builtins\Playground\index.js:9:1]
  9 │ import React, { useContext } from 'react';
 10 │ import { ErrorBoundary } from 'react-error-boundary';
 11 │ import { ThemeAntVContext } from "../../context";
 12 │ import { CodeRunner } from "dumi/theme/slots/CodeRunner";
    ·                            ─────────────────────────────
 13 │ import styles from "./index.module.less";
 14 │ /**
 15 │  * Markdown 标签插件 Playground
    ╰────

fatal - [Error: Build with mako failed.]
fatal - A complete log of this run can be found in:
● Mako ▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨▨ (26%) before resolve before resolve                                                                                                 fatal - E:\temp\G6-5.0.19\packages\site\node_modules\.cache\logger\umi.log
fatal - Consider reporting a GitHub issue on https://github.com/umijs/dumi/issues
 ELIFECYCLE  Command failed with exit code 1.

**Reply by Aarebecca** [reply]
暂没有遇到过你描述的问题
